justfile

```diff
@@ -19,14 +21,24 @@ venv:
     pip3 install -r requirements.txt
     source .venv/bin/activate
 
-
 # Run shell in docker
 run:
   docker run {{platform}} -v $(pwd):/rna/r2dt -v ./1.4:/rna/r2dt/data/cms -it --rm {{image}}
 
+# Run all tests in docker
+test-all:
+  docker run {{platform}} --rm -it -v ./:/rna/r2dt/ -v ./1.4/:/rna/r2dt/data/cms {{image}} bash -c "cd r2dt && ./r2dt.py test"
```

Comment:
I set the WORKDIR to `/rna/r2dt` in the main Dockerfile for consistency with previous versions so no need to do `cd r2dt` anymore